nectar/Cargo.toml

```diff
@@ -0,0 +1,68 @@
+[package]
+name = "nectar"
+version = "0.1.0"
+authors = ["CoBloX Team <team@coblox.tech>"]
+edition = "2018"
+
+[dependencies]
+anyhow = "1"
+async-trait = "0.1"
+bitcoin = { version = "0.23", features = ["rand", "use-serde"] }
+chrono = "0.4"
+clarity = "0.1"
+comit = { git = "https://github.com/comit-network/comit-rs/", package = "comit", rev = "76854090bb542ab55c5649c98ad1bc887bf5c764" }
```

Comment:
This should be a path dependency now :)

Reply:
was worried it wouldn't work when pointing to the latest comit but seems to be fine :)